payments-core/src/main/java/com/stripe/android/payments/PaymentFlowResultProcessor.kt

```diff
@@ -19,7 +19,7 @@ import kotlin.coroutines.CoroutineContext
 /**
  * Class responsible for processing the result of a [PaymentController] confirm operation.
  */
-internal sealed class PaymentFlowResultProcessor<T : StripeIntent, out S : StripeIntentResult<T>>(
+sealed class PaymentFlowResultProcessor<T : StripeIntent, out S : StripeIntentResult<T>>(
```

Comment:
This too if possible to inject into the PaymentSheetPaymentController would not need to be made public.